docs/guides/mssqlserver/pitr/archiver.md

```diff
@@ -0,0 +1,600 @@
+  fullBackup:
+    driver: WalG
+    task:
+      params:
+        databases: demo
+    scheduler:
+      successfulJobsHistoryLimit: 1
+      failedJobsHistoryLimit: 1
+      schedule: "/30 * * * *"
+    sessionHistoryLimit: 2
+    jobTemplate:
+      spec:
+        securityContext:
+          runAsUser: 0
+  walBackup:
+    runtimeSettings:
+      pod:
+        securityContext:
+          runAsUser: 0
+      container:
+        securityContext:
+          runAsUser: 0
+  backupStorage:
+    ref:
+      name: gcs-storage
+      namespace: demo
+```
+
+Let’s create the above `MSSQLServerArchiver`,
+
+```bash
+$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mssqlserver/pitr/examples/sample-mssqlserverarchiver.yaml
+mssqlserverarchiver.archiver.kubedb.com/sample-mssqlserverarchiver created
+```
+
+### Deploy Sample Microsoft SQL Server Database
+
+First, an `Issuer/ClusterIssuer` needs to be created, even if TLS is not enabled for `Microsoft SQL Server`. The issuer will be used to configure the TLS-enabled `Wal-G proxy server`, which is required for the SQL Server backup and restore operations.
+
+**Create Issuer/ClusterIssuer:**
+
+Now, we are going to create an example `Issuer` CR that will be used throughout the duration of this tutorial. Alternatively, you can follow this [cert-manager](https://cert-manager.io/docs/configuration/ca/) tutorial to create your own `Issuer`.
+
+By following the below steps, we are going to create our desired issuer,
+
+- Start off by generating our ca-certificates using openssl,
+
+```bash
+$ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./ca.key -out ./ca.crt -subj "/CN=mssqlserver/O=kubedb"
+```
+
+- create a secret using the certificate files we have just generated,
+
+```bash
+$ kubectl create secret tls mssqlserver-ca --cert=ca.crt  --key=ca.key --namespace=demo 
+secret/mssqlserver-ca created
+```
+
+Now, we are going to create an `Issuer` CR using the `mssqlserver-ca` secret that contains the ca-certificate we have just created. Below is the YAML of the `Issuer` CR that we are going to create,
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+ name: mssqlserver-ca-issuer
+ namespace: demo
+spec:
+ ca:
+   secretName: mssqlserver-ca
+```
+
+Let’s create the `Issuer` CR we have shown above,
+
+```bash
+$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mssqlserver/pitr/examples/mssqlserver-ca-issuer.yaml
+issuer.cert-manager.io/mssqlserver-ca-issuer.yaml created
+```
+
+**Create MSSQLServer CR:**
+
+Below is the YAML of a sample `MSSQLServer` CR that we are going to create for this tutorial:
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: MSSQLServer
+metadata:
+  name: sample-mssqlserver-ag
+  namespace: demo
+  labels:
+    archiver: "true"
+spec:
+  healthChecker:
+    timeoutSeconds: 100
+  archiver:
+    ref:
+      name: sample-mssqlserverarchiver
+      namespace: demo
```

Comment:
Tell that , We dont have to specify this field.  The archiver will be auto-selected using the `archiver.spec.databases` field.